Storage/SQL/JoinedHistoryVisitsTable.swift

```diff
@@ -139,9 +139,6 @@ class JoinedHistoryVisitsTable: Table {
             site.icon = icon
         }
 
-        let dt2 = (NSDate().timeIntervalSince1970 - visit.date.timeIntervalSince1970) / 86400
-        println("DT \(site.url): \(dt2)")
-
```

Comment:
Heh, this is definitely a tangent. It'd be helpful if you could split up these commits a little more in the future -- it's harder to read PRs with so many unrelated changes.